lib/index.js

```diff
@@ -6,14 +6,13 @@ var cache = new Map();
 
 
 /**
- * Mocks a method of an object.
+ * Mocks a value of an object.
  *
  * @param {Object} obj
  * @param {string} key
- * @param {!Function} method
+ * @param {!Function|Object} value
  */
-var method = module.exports = (obj, key, method) => {
-  method = method === undefined ? () => {} : method;
```

Comment:
Why delete this line?

Reply:
The mocked property could be anything, even undefined. When the library was first made, it was only meant to mock functions in objects, that's why this line was first added. But gradually it changed to mocking any type of property

Reply:
But it's a break change, should be published on 2.0.0

Reply:
Oh! Good point ._.